Modules/Microsoft365DSC/DSCResources/MSFT_SCRoleGroup/MSFT_SCRoleGroup.schema.mof

```diff
@@ -2,6 +2,7 @@
 class MSFT_SCRoleGroup : OMI_BaseResource
 {
     [Key, Description("The Name parameter specifies the name of the role. The maximum length of the name is 64 characters.")] String Name;
+    [Write, Description("The DisplayName parameter specifies the name of the role. The maximum length of the name is 256 characters.")] String DisplayName;
```

Comment:
Let's use the official description:

The DisplayName parameter specifies the friendly name of the role group. If the name contains spaces, enclose the name in quotation marks. This parameter has a maximum length of 256 characters..